Восьмая лабораторная работа выполнена

PopovMatvey opened the pull request:
Были выполнены все задания с 1-8:
1) 1-5 задания выполнены самостоятельно;
2) в шестом задании была создана кнопка, согласно требованиям;
3) в седьмом задании выполнена реализация "слоёного блока" тремя способами;
4) в восьмом задании была переделана реализация "слайд бара" под flex и grid.

PopovMatvey committed:
Create task-06.css

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Восьмая лабораторная работа выполнена

Были выполнены все задания с 1-8:
1) 1-5 задания выполнены самостоятельно;
2) в шестом задании была создана кнопка, согласно требованиям;
3) в седьмом задании выполнена реализация "слоёного блока" тремя способами;
4) в восьмом задании была переделана реализация "слайдбара" под flex и grid/

djilua1 commented on src/ex8_html-css-fundamentals-part1/task-06.html:
```diff
@@ -0,0 +1,20 @@
+    <!-- <button class="checkout-button">checkout</button> -->
+
+
+
```
Удалить лишние строчки + комментарий.

djilua1 commented on src/ex8_html-css-fundamentals-part1/css/task-06.css:
```diff
@@ -0,0 +1,39 @@
+.checkout-button {
```
При наведении на кнопку стрелка должна меняться на пальчик.

djilua1 commented on src/ex8_html-css-fundamentals-part1/css/task-07.css:
```diff
@@ -0,0 +1,83 @@
+h2 {
```
Слишком много общих стилей, можно в один класс их запихнуть.

djilua1 commented on src/ex8_html-css-fundamentals-part1/css/task-flex.css:
```diff
@@ -0,0 +1,37 @@
+aside {
+    background: pink;
+    width: 25%;
+    order: 999;
```
Плохая практика.

djilua1 commented on src/ex8_html-css-fundamentals-part1/css/task-flex.css:
```diff
@@ -0,0 +1,37 @@
+
+header, article, section, .module {
+    padding: 1rem;
+}
```
У меня вот так вот отобразилось:
![image](https://user-images.githubusercontent.com/43634050/162757839-39fcd918-1d0b-49b1-b252-cfa45480f2fd.png)

djilua1 commented on src/ex8_html-css-fundamentals-part1/css/task-grid.css:
```diff
@@ -0,0 +1,33 @@
+
+header, article, section, .module {
+    padding: 1rem;
+}
```
Тоже отобразилось неправильно.

PopovMatvey committed:
Изменённый css

PopovMatvey committed:
Исправления по восьмой лабораторной работе

Были исправлены все шесть замечаний:
1) Был удалён комментарий и лишние строчки в task-06.htm;
2) При наведении на кнопку стрелка меняется на пальчик в шестом задании;
3) Общие стили были вынесены в отдельные классы;
4) Плохая практика была устранена task-flex.css;
5 и 6) В восьмом задании и для реализации flex, и для реализации grid было поправлено отображение.
